salt/modules/chocolatey.py

```diff
@@ -248,11 +248,11 @@ def list_(narrow=None,
     if narrow:
         cmd.append(narrow)
     if salt.utils.is_true(all_versions):
-        cmd.append('-allversions')
+        cmd.append('--allversions')
     if salt.utils.is_true(pre_versions):
-        cmd.append('-prerelease')
+        cmd.append('--prerelease')
     if source:
-        cmd.extend(['-source', source])
+        cmd.extend(['--source', source])
     if local_only:
```

Comment:
It also appears I missed these two, which is dumb because I added the second line!